docs/admin-manual/custom-authentication.md

```diff
@@ -3,6 +3,12 @@ title: Custom authentication
 sidebar_position: 40
 ---
 
+:::warning
+
+Since LibreTime v3.0.0-alpha.13, this documentation is out of date, as it rely on the Apache2 web server and the default web server installed by LibreTime is NGINX.
```

Comment:
```suggestion
Since LibreTime v3.0.0-alpha.13, this documentation is out of date, as it relies on the Apache2 web server and the default web server installed by LibreTime is now NGINX.
```